application/src/test/java/org/opentripplanner/apis/vectortiles/DebugVectorTilesResourceTest.java

```diff
@@ -0,0 +1,51 @@
+package org.opentripplanner.apis.vectortiles;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.TestServerContext;
+import org.opentripplanner.apis.vectortiles.model.LayerType;
+import org.opentripplanner.ext.fares.impl.DefaultFareService;
+import org.opentripplanner.inspector.vector.LayerParameters;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.test.support.HttpForTest;
+import org.opentripplanner.transit.service.TimetableRepository;
+
+class DebugVectorTilesResourceTest {
+
+  private record LayerParams(String name, LayerType type) implements LayerParameters<LayerType> {
+    @Override
+    public String mapper() {
+      return "type";
+    }
+  }
+
+  private static final List<LayerParameters<LayerType>> LAYERS = List.of(
+    new LayerParams("edge", LayerType.Edge),
+    new LayerParams("areaStop", LayerType.AreaStop)
+  );
+
+  @Test
+  void tileJson() {
+    var resource = new DebugVectorTilesResource(
+      TestServerContext.createServerContext(
+        new Graph(),
+        new TimetableRepository(),
+        new DefaultFareService()
```

Comment:
Btw it would useful to be able to create server context without fare service. In tests it's rarely needed.

Reply:
I think it would be useful. I believe that a builder would be appropriate here.